bootstrap/roles/appliance-build.bootstrap/tasks/main.yml

```diff
@@ -19,31 +19,94 @@
   register: toplevel
 
 - apt:
-    name: aptitude
+    name: software-properties-common
```

Comment:
Why do we need this?

Reply:
To install the ansible PPA. We add it currently in the docker container, so this isn't changing anything.

Reply:
We should probably add a comment.

Reply:
I'll actually remove this, and the adding of the PPA, since we only support building on the bootstrap image, and that already has ansible installed.